fdbserver/KeyValueStoreShardedRocksDB.actor.cpp

```diff
@@ -672,7 +672,8 @@ class ShardManager {
 						                              rocksdb::DB::Properties::kCompressionRatioAtLevelPrefix +
 						                                  std::to_string(it->level),
 						                              &propValue));
-						e.detail("Level" + std::to_string(it->level), std::to_string(it->size) + " " + propValue);
+						e.detail("Level" + std::to_string(it->level),
```

Comment:
Rocksdb has "LevelStats" property which gives Level, files, size. KeyValueStoreRocksDB is using this. If you are looking something like below, better use this property.

Prints something like this: https://splunk-if.icloud.apple.com/en-US/app/search/search?sid=1680620973.40426_139BDE67-98A9-4FD7-A5EE-800DC055E245